src/bp-moderation/screens/single/admin/report-single.php

```diff
@@ -190,42 +191,48 @@
 										}
 										?>
 										<a href="javascript:void(0);"
-										class="button button-primary bp-hide-request single-report-btn"
-										data-id="<?php echo esc_attr( $moderation_request_data->item_id ); ?>"
-										data-type="<?php echo esc_attr( $moderation_request_data->item_type ); ?>"
-										data-nonce="<?php echo esc_attr( wp_create_nonce( 'bp-hide-unhide-moderation' ) ); ?>"
-										data-action="<?php echo esc_attr( $action_type ); ?>"
-										title="<?php echo esc_html( $action_label ); ?>">
+                                           class="button button-primary bp-hide-request single-report-btn"
```

Comment:
@janakkaneriya007 Please reformat the code using PHPCS. This should not change. you need to reformate code only you adding. it's very hard to review